File: src/org/ggp/base/util/gdl/grammar/GdlVariable.java
package org.ggp.base.util.gdl.grammar;

class GdlVariable(GdlTerm):
{

    name = ''

    GdlVariable(String name)
	{
        this.name = name.intern();

    def String getName()
	{
        return name;

    def boolean isGround()
	{
        return false;

    def GdlSentence toSentence()
	{
        throw new RuntimeException("Unable to convert a GdlVariable to a GdlSentence!");

    def String toString()
	{
        return name;

File: ggpy/cruft/autocode/GdlVariable.py
#!/usr/bin/env python
""" generated source for module GdlVariable """
# package: org.ggp.base.util.gdl.grammar
@SuppressWarnings("serial")
class GdlVariable(GdlTerm):
    """ generated source for class GdlVariable """
    name = str()

    def __init__(self, name):
        """ generated source for method __init__ """
        super(GdlVariable, self).__init__()
        self.name = name.intern()

    def getName(self):
        """ generated source for method getName """
        return self.name

    def isGround(self):
        """ generated source for method isGround """
        return False

    def toSentence(self):
        """ generated source for method toSentence """
        raise RuntimeException("Unable to convert a GdlVariable to a GdlSentence!")

    def __str__(self):
        """ generated source for method toString """
        return self.name

